tests/SendSalesCommandTest.php

```diff
@@ -159,6 +166,7 @@ function ($notification, $channels, $notifiable) {
         config([
             'trx_mall_upload_sales_data_api.notifications.mail.name' => $this->mailConfig['name'],
             'trx_mall_upload_sales_data_api.notifications.mail.email' => $this->mailConfig['email'],
+            'trx_mall_upload_sales_data_api.notifications.mail.enable_failure_notifications_only' => false,
```

Comment:
Do we need to mention?

Reply:
Actually, I added there, in case, if new test gets added later in the same group, setting the flag won't be needed.